packages/google_sign_in/google_sign_in_web/lib/google_sign_in_web.dart

```diff
@@ -259,14 +259,14 @@ class GoogleSignInPlugin extends GoogleSignInPlatform {
   Future<void> signOut() async {
     await initialized;
 
-    _gisClient.signOut();
+    await _gisClient.signOut();
```

Comment:
These seemed like bugs, since all the similar calls around it that return something do await. @ditman ?

Reply:
LGTM, I don't know why this was sneakily unawaited, probably my mistake!